docs/develop/sdk/1-dev-setup.md

```diff
@@ -0,0 +1,150 @@
+```
+
+You should see output similar to the following:
+
+```
+2022-05-05 13:25:12 KILT Node
+2022-05-05 13:25:12 ✌️  version 1.6.2
+2022-05-05 13:25:12 ❤️  by KILT <info@kilt.io>, 2019-2022
+2022-05-05 13:25:12 📋 Chain specification: Development
+2022-05-05 13:25:12 🏷  Node name: subdued-chair-0035
+2022-05-05 13:25:12 👤 Role: AUTHORITY
+2022-05-05 13:25:12 💾 Database: RocksDb at /tmp/substrateufCNUV/chains/development/db/full
+2022-05-05 13:25:12 ⛓  Native runtime: mashnet-node-10620 (mashnet-node-0.tx3.au4)
+2022-05-05 13:25:13 🔨 Initializing Genesis block/state (state: 0xb4a2…94b3, header-hash: 0x09fc…3a2b)
+2022-05-05 13:25:13 👴 Loading GRANDPA authority set from genesis on what appears to be first startup.
+2022-05-05 13:25:14 Using default protocol ID "sup" because none is configured in the chain specs
+2022-05-05 13:25:14 🏷  Local node identity is: 12D3KooWMCqWaxXTQbmG9feCe4cMzjCzUKfm5T6VvGDmh8X5QHe9
+2022-05-05 13:25:14 📦 Highest known block at #0
+2022-05-05 13:25:14 〽️ Prometheus exporter started at 127.0.0.1:9615
+2022-05-05 13:25:14 Listening for new connections on 0.0.0.0:9944.
+2022-05-05 13:25:19 💤 Idle (0 peers), best: #0 (0x09fc…3a2b), finalized #0 (0x09fc…3a2b), ⬇ 0 ⬆ 0
+2022-05-05 13:25:20 Accepted a new tcp connection from 172.17.0.1:56636.
+2022-05-05 13:25:23 🙌 Starting consensus session on top of parent 0x...
+2022-05-05 13:25:23 🎁 Prepared block for proposing at 1 (3 ms) [hash: 0x...; parent_hash: 0x09fc…3a2b; extrinsics (1): [0xae1a…0701]]
+2022-05-05 13:25:23 🔖 Pre-sealed block for proposal at 1. Hash now 0x..., previously 0x....
+```
+
+Congratulations!
+You are running your own blockchain. 🎉
+
+The blockchain exposes a websocket endpoint on port `9944` and an RPC endpoint on port `9933`.
+You can test that by first calling an RPC endpoint using curl.
+
+```
+curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "system_name", "params":[]}' http://127.0.0.1:9933/
+```
+
+This should give you `{"jsonrpc":"2.0","result":"KILT Node","id":1}` as a response.
+
+But you can also connect to the websocket endpoints using our little script above.
+Simple replace the peregrine address `wss://peregrine.kilt.io/parachain-public-ws` with `ws://127.0.0.1:9944`.
+Note that we connect to the port `9944` as we are using the websocket protocol for our SDK and not bare http.
+
+
+## Standalone vs. Parachain (Peregrine testnet/Spiritnet mainnet)
+
+The standalone chain is very close in functionality to our parachains but there are a few fundamental differences between them.
+
+<!-- ### Blocktime
+
+Block time is actually the same, but this might change in the future. -->
+
+### Governance
+
+While governance is a very important part of our parachains, it is not used in the standalone version and replaced by the sudo pallet.
+Any of the following pallets are not part of the standalone chain, but part of the parachain runtime:
```

Comment:
```suggestion
None of the following pallets is part of the standalone chain, but they are all part of the parachain runtime:
```

Reply:
Isn't it `are` since there are multiple pallets?
```suggestion
None of the following pallets are part of the standalone chain, but they are all part of the parachain runtime:
```

Reply:
I think you are right 👍